docs/01-app/03-api-reference/05-config/01-next-config-js/useLightningcss.mdx

```diff
@@ -4,14 +4,18 @@ description: Enable experimental support for Lightning CSS.
 version: experimental
 ---
 
-Experimental support for using [Lightning CSS](https://lightningcss.dev), a fast CSS bundler and minifier, written in Rust.
+Experimental support for using [Lightning CSS](https://lightningcss.dev) with webpack. Lightning CSS is a fast CSS transformer and minifier, written in Rust.
+
+If this option is not set, Next.js on webpack uses [PostCSS](https://postcss.org/) with [`postcss-preset-env`](https://www.npmjs.com/package/postcss-preset-env) by default.
+
+Turbopack uses Lightning CSS by default since Next 14.2. This configuration option has no effect on Turbopack. Turbopack always uses Lightning CSS.
 
 ```ts filename="next.config.ts" switcher
 import type { NextConfig } from 'next'
 
 const nextConfig: NextConfig = {
   experimental: {
-    useLightningcss: true,
+    useLightningcss: false, // default, ignored on Turbopack
```

Comment:
can you add something to turbopack-warnings.ts for this one. looks like we don't warn about it (should be sufficient to just add `experimental.useLightningcss` to the array at the top